core/sqf/src/seatrans/hbase-trx/src/main/java/org/apache/hadoop/hbase/coprocessor/transactional/generated/TrxRegionProtos.java

```diff
@@ -26558,6 +26558,16 @@ public Builder clearHasException() {
      * <code>required int64 auditSeqNum = 4;</code>
      */
     long getAuditSeqNum();
+
+    // required bool ageCommitted = 5;
+    /**
+     * <code>required bool ageCommitted = 5;</code>
+     */
+    boolean hasAgeCommitted();
+    /**
+     * <code>required bool ageCommitted = 5;</code>
```

Comment:
What is the point of the repeated <code>required bool ageCommitted = 5;</code> comments? There seems to be several of them.